sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -312,7 +322,7 @@ class EthService(val scNodeViewHolderRef: ActorRef, val nvtimeout: FiniteDuratio
   @RpcMethod("eth_gasPrice")
   def gasPrice: Quantity = {
     applyOnAccountView { nodeView =>
-      getStateViewAtTag(nodeView, "latest") { tagStateView => new Quantity(tagStateView.getBaseFee) }
+      getBlockAtTag(nodeView, "latest") { block => new Quantity(block.header.baseFee)}
```

Comment:
So here we want to predict the min gasPrice for given point.
For example, "latest", it seems we need to return not the block baseFee, but the next potential block baseFee.
We may use `def calculateBaseFee(nodeView: View, parentId: BlockId)`

Reply:
afaik we will need this + the result of SuggestTipCap - so we should be fine within this PR, no?